CHANGELOG.react.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 -   Fix stroke-width property on react progress bar.
 -   Fixes to Text Area related to Input resizing and `valid` and `error` states in Material
+-   Make `Switch` a controlled component
```

Comment:
This is a breaking change even though no one might be using this component right now ^^.
We usually mark breaking change with `_[BREAKING]_`

Reply:
True ! Will update.